src/libraries/Common/tests/TestUtilities/System/IO/FileCleanupTestBase.cs

```diff
@@ -14,11 +15,9 @@ namespace System.IO
     /// <summary>Base class for test classes the use temporary files that need to be cleaned up.</summary>
     public abstract partial class FileCleanupTestBase : IDisposable
     {
-        private static readonly Lazy<bool> s_isElevated = new Lazy<bool>(() => AdminHelpers.IsProcessElevated());
-
         private string fallbackGuid = Guid.NewGuid().ToString("N").Substring(0, 10);
 
-        protected static bool IsProcessElevated => s_isElevated.Value;
+        protected static bool IsProcessElevated => Environment.IsPrivilegedProcess;
```

Comment:
This file is being used by projects that target earlier framework versions and cannot use `Environment.IsPrivilegedProcess`. What you could do is modify `AdminHelpers.IsProcessElevated to use it, and maybe cache the value for the earlier frameworks there.

Reply:
AdminHelpers.IsProcessElevated is used to test Environment.IsPrivilegedProcess

Reply:
Hmm, so I guess you can't change it here.